### ISSUE TITLE
Zlinky_TIC - Regular reading of the "Currentindextarif" attribute

### ISSUE THREAD

**Issue by Gouns68** [Koenkk/zigbee2mqtt#21932]
### What happened?

Hello,
I'm on Jeedom and I've migrated to the Z2M-based plugin (version 1.36.0). I have a problem with my zlinky_tic module (firmware 4000-0013) in terms of the speed at which changes are sent. Z2M sends all the values several times a minute via mqtt even if they don't change. Current, voltage and counter values are updated every minute with the new values. On the other hand, NTARF and LTARF values take several minutes (5 to 10) to go up via mqtt at the time of tariff changes, which take place at around 10pm and 6am (which poses a problem for causing rapid load shedding after a change in the energy supplier's tariff). I didn't have this problem with the old Jeedom Zeebee plugin with the same zlinky_tic module. Does anyone know how to make the values of these 2 elements increase at the same rate as the voltages, currents and meters? Thank you very much.

### Zigbee2MQTT version

1.36.0

### Adapter firmware version

4000-0013

### Adapter

zlinky_tic

### Setup

Jeedom

**Comment by Gouns68** [comment]
By analysing the Z2M logs, I can see that values such as "apparentPower" are transmitted every minute, whereas the values I'm interested "currentIndexTarif" (00:50:20) are transmitted every 10 minutes.
You might think that it's the ZLinky_Tic module that only sends at this rate, but I think I read somewhere a comment from the manufacturer LiXee that it only sends on request, which is what Z2M would ask for.

<img width="1579" alt="A" src="https://github.com/Koenkk/zigbee2mqtt/assets/139239167/020f631d-9f6a-4e09-b70a-42f75895174a">
<img width="1579" alt="B" src="https://github.com/Koenkk/zigbee2mqtt/assets/139239167/058acd96-cddf-4b7b-a9ed-c454108b9a7e">

**Comment by Gouns68** [comment]
I assumed that I had to add "repporting" for the "liXeePrivate" cluster and the "currenIndexTarif" attribute, but when I applied, I systematically got the error: "'UNREPORTABLE_ATTRIBUTE')'"
<img width="1579" alt="C" src="https://github.com/Koenkk/zigbee2mqtt/assets/139239167/8f6b0050-685c-4735-8d70-0c6339203b71">

However, if I use "Dev console" with the same "Endpoint", "Cluster" and "attribute", the "READ" sends me back a reading of my value. 
<img width="1249" alt="D" src="https://github.com/Koenkk/zigbee2mqtt/assets/139239167/0f20e664-15d6-4e27-bc25-eb1a1672dced">

I think I've reached the end of what my knowledge can do, so if some kind soul could help me retrieve this value regularly (every minute), that would be a great help.

**Comment by Gouns68** [comment]
In the end, it's not a discussion but a monologue.... I did a number of tests by changing the "measurement pool interval" to a different value and I restarted Zigbee2MQTT each time and it seems that I've solved the problem because now all the values are updated at least every 60 seconds. At first I was afraid, because after restarting Zigbee2MQTT the module systematically sends a lot of data for a few minutes, almost every second (whatever the value of the pool), but finally after a few minutes things stabilised at a regular rate. I also noticed that at each start-up, Z2M returns several errors concerning the Zlinky_TIC. I'm not sure what's causing this..... 
`error 2024-03-25 13:02:58: Failed to configure '0x00158d0006a402c5', attempt 1 (Error: ConfigureReporting 0x00158d0006a402c5/1 haElectricalMeasurement([{"attribute":"totalReactivePower","minimumReportInterval":60,"maximumReportInterval":900,"reportableChange":1}], {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (Timeout - 63381 - 1 - 16 - 2820 - 7 after 10000ms)`
